manifests/n/NetEase/IdentityV/2.0.0.1/NetEase.IdentityV.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: NetEase.IdentityV
+PackageVersion: 2.0.0.1
```

```suggestion
PackageVersion: 2.0.0.1
MinimumOSVersion: 10.0.0.0
```

- Add missing property for keeping consistency.